src/screens/SearchResultsScreen.tsx

```diff
@@ -20,12 +23,39 @@ export async function loader({request}: {request: Request} ): Promise<string> {
  */
 export default function SearchResultsScreen() {
 	const query: string = useLoaderData() as string;
+	const location = useLocation();
```

Comment:
We do not need to useLocation here. The query on line 25 is already the users search string